www/main/workspace-bar/components/WorkspaceBar.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { remote } from 'electron';
+
+const { store } = remote.require('./app');
+
+const WorkspaceBar = () => {
+  const workspaces = ['Presenter', 'Single Screen'];
```

Comment:
Should be translated.

Reply:
Can we also change "Presenter" to "Presentation", and "Single Screen" to "Single Display"